cms/tests/test_app_registration.py

```diff
@@ -53,4 +53,186 @@ def test_imports_cms_apps_files(self):
     ])
     def test_raises_exception_raised_in_cms_file(self):
         with self.assertRaises(KeyError):
-            autodiscover_cms_files()
+            app_registration.autodiscover_cms_configs()
+
+    @override_settings(INSTALLED_APPS=[
+        'cms.tests.test_app_registry.app_with_cms_extension',
+        'cms.tests.test_app_registry.app_without_cms_file'
+    ])
+    def test_adds_cms_app_attribute_to_django_app_config(self):
```

Comment:
I would personally add a small comment on what each test is looking for no matter how obvious it seems to you. I had to look at it a few times to follow it.